tests/storages_tests/test_storages.py

```diff
@@ -1385,3 +1386,27 @@ def test_retry_failed_trial_callback_repetitive_failure(storage_mode: str) -> No
         assert "retry_history" not in trials[4].system_attrs
         assert trials[5].system_attrs["failed_trial"] == 4
         assert trials[5].system_attrs["retry_history"] == [4]
+
+
+@pytest.mark.parametrize("storage_mode", STORAGE_MODES)
+def test_pickle_storage(storage_mode: str) -> None:
+    if "redis" in storage_mode:
+        pytest.skip("Redis storage is not picklable")
```

Comment:
[Nit] full sentence looks better.

```suggestion
        pytest.skip("Redis storage is not picklable.")
```